packages/react/src/components/Forms/Examples/ReactHookForm/Sports.stories.tsx

```diff
@@ -34,7 +34,11 @@ const sportsSchema = Yup.object().shape({
   rewards: Yup.array().of(Yup.string().defined()).defined(),
 });
 
-export const Sports = () => {
+export interface SportProps {
+  values?: SportsStoryFormValues;
+}
+
+export const Sports = ({ values }: SportProps) => {
```

Comment:
The form crashes on submit  https://www.loom.com/share/c7bb36e44fc64778be4db099eb8c3509

Reply:
both Sport and SportEdit stories have been place in the same file, it fix this weird behavior of storybook.